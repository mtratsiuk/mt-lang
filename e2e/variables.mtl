(def myConst 10)

(def myFunc
  (|p|
    (def local 5)
    (def sub
      (-
        p
        local))

    (- myConst sub)))

(print (myFunc 20))
