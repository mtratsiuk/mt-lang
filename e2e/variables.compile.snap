const myConst = 10;

const myFunc = ((p) => {
  const local = 5;
  const sub = (p - local);
  return (myConst - sub);
});

console.log(String(myFunc(20)));
